test/Conversion/TorchOnnxToTorch/simple_ops_g_to_p.mlir

```diff
@@ -4,359 +4,13 @@
 // level constants. This is a pragmatic choice which lets us have a lot
 // of tests in this file, whereas the others tend to be more bespoke.
 
-// CHECK-LABEL: func.func @test_greater
```

Comment:
Please don't delete these tests. These test are useful for capturing the correctness of our other OnnxToTorch lowerings.

Reply:
I am going to remove this complete PR for now.